packages/react/src/alert-dialog/backdrop/AlertDialogBackdropCssVars.ts

```diff
@@ -0,0 +1,7 @@
+export enum AlertDialogBackdropCssVars {
+  /**
+   * Indicates the client height of the document body.
```

Comment:
Let's explain why this is useful. I'd intuitively just use `position: fixed` and ignore this.